packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -82,6 +83,43 @@ function Example() {
 }
 ```
 
+## Long press
+By default, a MenuTrigger's Menu is opened by pressing the trigger element or activating it via the <Keyboard>Space</Keyboard> or <Keyboard>Enter</Keyboard> keys. However, there may be cases in which your trigger element
+should perform a separate default action on press such as selection, and should only display the Menu when long pressed. This behavior can be changed by providing `"longPress"` to the `trigger` prop. With this prop, the Menu will only be opened upon
+pressing and holding the trigger element or by using the <Keyboard>Option</Keyboard> (<Keyboard>Alt</Keyboard> on Windows) + <Keyboard>Down Arrow</Keyboard> keys while focusing the trigger element.
```

Comment:
I noticed that `option + up arrow` also triggers a long press, but the focus is on the last element in the menu.

Reply:
yep, i think we agreed it was overkill to explain that when the docs already mention the focus strategy